ogusa/default_parameters.json

```diff
@@ -185,11 +185,20 @@
         "description": "Annual rate of time preference for households.",
         "section_1": "Household Parameters",
         "section_2": "Behavioral Assumptions",
-        "notes": "Default value from Carroll (JME, 2009).",
+        "notes": "Default value from Carroll (JME, 2009).  Allows for each type to have a different value.",
         "type": "float",
+        "number_dims": 1,
         "value": [
             {
-                "value": 0.96
+                "value": [
```

Comment:
Did you have a paper in mind for assigning different defaults by j? Or is the idea to calibrate them as part of OG-USA to better match wealth and MPC distributions?

I'm doing separate research that involves time preference across the income distribution, so this PR caught my eye. I didn't see it in Carroll (2017) but only gave it a cursory look.